src/app/component-library/catalog/asset-detail-dialog/asset-property-grid-group-builder.ts

```diff
@@ -88,6 +93,33 @@ export class AssetPropertyGridGroupBuilder {
       fields.push(...this.buildMdsProperties(asset, true));
     }
 
+    if (
+      asset.httpProxyMethod != null ||
+      asset.httpProxyBody != null ||
+      asset.httpProxyPath != null ||
+      asset.httpProxyQueryParams != null
+    ) {
+      let showDetailsObject = {
+        httpProxyMethod: asset.httpProxyMethod ? 'Enabled' : 'Disabled',
+        httpProxyBody: asset.httpProxyBody ? 'Enabled' : 'Disabled',
+        httpProxyPath: asset.httpProxyPath ? 'Enabled' : 'Disabled',
+        httpProxiedQueryParams: asset.httpProxyQueryParams
```

Comment:
const proxyDetails = [
    'httpProxyMethod', 
    'httpProxyBody', 
    'httpProxyPath', 
    'httpProxyQueryParams'
]; anyway to use an array like this to check both conditions?